mangadex_downloader/utils.py

```
@@ -230,6 +230,10 @@ def submit(self, job, blocking=True):
 
         return fut.result()
 
+    def wait(self, timeout=None):
```

Ruff (0.13.2)

Unused method argument: `timeout`

(ARG002)